docs/getting-started/security-ui.asciidoc

```diff
@@ -22,204 +23,69 @@ image::images/search-bar.png[]
-
-[float]
-[[detection-engine-ui]]
-=== Alerts page
-
-The Alerts page allows you to view and manage all alerts to monitor activity within your network. Refer to <<detection-engine-overview, Detections and Alerts>> for more information.
+<<detection-engine-overview, *Alerts*>>: View and manage alerts to monitor activity within your network.
 
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
-[float]
-=== Findings page
-
-The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
+<<findings-page, *Findings*>>: Review your Kubernetes infrastructure’s performance on a range of security benchmarks.
 
 [role="screenshot"]
 image::cloud-native-security/images/findings-page.png[The Findings page]
 
-[float]
-[[rules-page]]
-=== Rules page
-
-The Rules page allows you to view and manage all detection rules. Refer to <<rules-ui-management, Manage detection rules>> for more information about prebuilt and custom rules.
+<<timelines-ui, *Timelines*>>: Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are responsive and allow you to share your findings among other team members.
 
 [role="screenshot"]
-image::detections/images/all-rules.png[Shows the Rules page]
+image::images/timeline-ui.png[Shows the Timeline page]
 
-[float]
-[[exceptions-page]]
-=== Exception lists page
+TIP: Select the collapsable *Timeline* button at the bottom of the {security-app} to start an investigation.
 
-The Exception lists page allows you to view and manage all rule exceptions. Refer to <<detections-ui-exceptions, Rule exceptions and value lists>> for more information about rule exceptions.
+<<cases-overview, *Cases*>>: Open and track security issues.
 
 [role="screenshot"]
-image::detections/images/exceptions-page.png[Shows the Exception lists page]
-
-[float]
-[[hosts-ui]]
-=== Hosts page
+image::cases/images/cases-home-page.png[Shows the Cases page]
+*Explore*: Expand this section to view the following pages:
 
-The Hosts page provides key metrics for host-related security events and a set of interactive data tables. Refer to <<hosts-overview, Hosts page>> for more information.
+* <<hosts-overview, *Hosts*>>: View key metrics for host-related security events through graphs, charts, and interactive data tables.
 
 [role="screenshot"]
 image::management/hosts/images/hosts-ov-pg.png[Shows the Hosts page]
 
-
-[float]
-[[network-page]]
-=== Network page
-
-The Network page provides key network activity metrics via an interactive map and network event tables
-that enable interaction with Timeline. Refer to <<network-page-overview, Network page overview>> for more information.
+* <<network-page-overview, *Network*>>: View key network activity metrics through an interactive map and network event tables that interact with Timeline.
```

Comment:
```suggestion
* <<network-page-overview, *Network*>>: View network event tables that interact with Timeline, and key network activity metrics on an interactive map.
```
To me the sentence is more readable in this order because there's less ambiguity about whether "network" is ever being used in its verb form.

Reply:
Amendment to @benironside's helpful edit above -- maybe less likely, but "key" could also be read as a verb, and I think it's a stronger structure to have clear verbs in each phrase:
```suggestion
* <<network-page-overview, *Network*>>: View network event tables that interact with Timeline, and visualize key network activity metrics on an interactive map.
```

Reply:
Are we visualizing activity metrics on the network map though? When I click around on the map, I get these brief, informational pop-ups that list a few key details about the host.
<img width="1422" alt="Screen Shot 2022-09-15 at 4 28 43 PM" src="https://user-images.githubusercontent.com/79124755/190502701-33b7cb8a-3df4-4daa-b90b-969c428736c6.png">

I also kind of want to tweak the bit about event tables that can interact with Timeline. How about something like the following:

```suggestion
* <<network-page-overview, *Network*>>: Explore the interactive map to discover key network activity metrics and investigate network events further in Timeline.
```